aip/auth/4122.md

```diff
@@ -0,0 +1,349 @@
+---
+id: 4122
+scope: auth
+state: reviewing
+created: 2020-12-15
+---
+
+# Service Account Impersonation
+
+A service account or a user identity in GCP can impersonate another service 
+account, and request short-lived credentials on behalf of that service account.
+
+The request can be direct or delegated.
+
+In the direct case, the caller is authenticated as either a Google account or a
+service account, and makes a direct request to create short-lived credentials on
+behalf of another service account. The requesting identity must have the Service
+Account Token Creator role (`roles/iam.serviceAccountTokenCreator`) on the target
+service account.
+
+In the delegated case, the caller is also authenticated as either a Google
```

Comment:
So is the direct case effectively just a special case of the delegated case, where the delegation chain is empty?